app.js

```diff
@@ -80,15 +82,15 @@ async function searchInFile({
 
   return (
     returnContent
-      ? { matches: [...fileContent.matchAll(parsedRegex)] }
+      ? { matches: [...fileContent.match(parsedRegex)] }
```

Comment:
why not `matchAll`?

Reply:
matchAll fails with global RegExp like /Kaholo/gi. If we are enabling the flags and using /gm by default, we need to use match() instead. I tested it - match() works with anything and we use /gm as our default if no flags are given.